owncloud/owncloud.py

```diff
@@ -731,6 +731,26 @@ def user_exists(self, user_name):
         :raises: ResponseError in case an HTTP error status was returned
 
         """
+        users=search_users(self, user_name)
+
+        if len(users) > 0:
+            for user in users:
+            	if user == user_name:
+            		return True
+
+            return False
+        else:
+        	return False
```

Comment:
The indent still looks off. Please use spaces instead of tabs (4 spaces for an indent). Maybe your editor used tabs.

Here I'd recommend only having a single `return False` at the end of the function instead of an else block.